content/md/assets/full-automatic-pam-migration.md

```diff
@@ -0,0 +1,112 @@
+---
+id: full-automatic-pam-migration
+title: The **full-automatic** migration guide for PAM assets
+popular: false
+ee-only: true
+related: from-PAM-to-new-AM, how-to-migrate-from-pam
+---
+
+# What's the full-automatic migration?
+In this migration, **everything is automated**, you just need to launch one single command and your PAM assets will be migrated in the 4.0 new Asset Manager:
+- One asset family will be created gathering all you assets.
+- All your assets will be migrated and linked to the new asset family.
+- All you existing PAM asset collection product attributes will be linked to the new asset family.
+- And all your links between assets and products / product models will be kept.
```

Comment:
maybe warn here that this will mean that all there assets will be mixed and split them by family will be hard later. And add that we provided this migration way as a simple and easy path but we advise them to take a look at other ways of doing it :)